expo/gbe/models.py

```diff
@@ -303,17 +303,18 @@ def get_shows(self):
                  for act in acts if act.accepted == 3 and act.is_current]
         return sum([list(s) for s in shows], [])
 
-    def get_schedule(self):
+    def get_schedule(self, conference=None):
```

Comment:
I can't imagine any case where we'd want to use the unparameterized schedule() function, and I can only find one place where we are using it. get_schedule and schedule should be one function, which takes a conference, and defaults to the current conference. (not to all conferences)

Since I can only find one place where the schedule() function is called (in printable_schedules) this should be a pretty trivial change.